site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -159,14 +188,37 @@ authentication-related functionalities.
 
 In the future, we hope to support more features, including:
 
-*  Automatic sign-in
-*  Authorization prompt
-*  Access and refresh tokens
-*  Front-channel logout: sign-out from the relying party (RP) initiated by
-   the identity provider (IdP)
-*  OpenID Connect (OIDC) session management
-*  Cross-origin iframes
-*  Personalized buttons
+Short-term:
```

Comment:
I noticed that there's some update in the "shipping" doc on this section. Could you sync the change here?